app/controllers/law_offices_controller.rb

```diff
@@ -0,0 +1,26 @@
+class LawOfficesController < ApplicationController
+  def index
+  end
+
+  def new
+    @law_office = Law_office.new
+  end
+  
+  def create
+  end
+  
+  def edit
+    @law_office = law_office.find(params[:id])
```

Comment:
In the cases where you want to do `.new` or `.find`, you'll want to call the method on the `LawOffice` class like this: `LawOffice.new`